aspnet/web-pages/overview/getting-started/introducing-aspnet-web-pages-2/getting-started.md

```diff
@@ -1,15 +1,15 @@
 ---
 uid: web-pages/overview/getting-started/introducing-aspnet-web-pages-2/getting-started
-title: "Getting Started | Microsoft Docs"
+title: Getting Started with ASP.NET | Microsoft Docs
```

Comment:
IF this doesn't resolve a warning, why are you making this change?

Reply:
This line actually flagged a duplicate-titles error in validation. The change proposed fixes the error.